src/nrf_802154_types.h

```diff
@@ -138,6 +138,15 @@ typedef uint8_t nrf_802154_ack_data_t;
 #define NRF_802154_ACK_DATA_PENDING_BIT 0x00
 #define NRF_802154_ACK_DATA_IE          0x01
 
+/**
+ * @brief Methods of source address matching.
+ */
+typedef uint8_t nrf_802154_src_addr_match_t;
+
+#define NRF_802154_SRC_ADDR_MATCH_THREAD   0x00 // !< Implementation for Thread protocol.
```

Comment:
```suggestion
#define NRF_802154_SRC_ADDR_MATCH_THREAD   0x00 // !< Implementation for the Thread protocol.
```
Are we talking here about Nordic's protocol? Then why not use `for Nordic's Thread protocol`? @hubertmis ?

Reply:
No, it's not Nordic's protocol. It's common specification for all Thread Group members.